driver/src/test/scala/X509DriverSpec.scala

```diff
@@ -0,0 +1,75 @@
+
+import org.specs2.concurrent.ExecutionEnv
+import reactivemongo.api.{ DefaultDB, X509Authentication }
+import reactivemongo.bson.BSONDocument
+import reactivemongo.core.actors.Exceptions.{ InternalState, PrimaryUnavailableException }
+import reactivemongo.core.nodeset.Authenticate
+import scala.concurrent.Await
+
+class X509DriverSpec(implicit ee: ExecutionEnv) extends org.specs2.mutable.Specification {
```

Comment:
Would rather execute with the tests existing in `DriverSpec`, with appropriate tags, maybe with a trait mixin